examples/real-world/webpack.config.js

```diff
@@ -14,7 +14,12 @@ module.exports = {
   },
   plugins: [
     new webpack.optimize.OccurenceOrderPlugin(),
-    new webpack.HotModuleReplacementPlugin()
+    new webpack.HotModuleReplacementPlugin(),
+    new webpack.DefinePlugin({
+      'process.env': {
+        NODE_ENV: JSON.stringify(process.env.NODE_ENV)
```

Comment:
Afaik this won't work since we don't define the `NODE_ENV` in the start command, so `process.env.NODE_ENV` will be undefined.